src/main/resources/application-preprod.yaml

```
@@ -126,23 +125,30 @@ no.nav.security.jwt:
 
 spring:
   datasource:
-    url: jdbc:postgresql://b27dbvl009.preprod.local:5432/familie-ba-sak
+    url: jdbc:postgresql://${DB_HOST}:${DB_PORT}/familie-ba-sak
+    username: ${DB_USERNAME}
+    password: ${DB_PASSWORD}
   kafka:
     bootstrap-servers: b27apvl00045.preprod.local:8443,b27apvl00046.preprod.local:8443,b27apvl00047.preprod.local:8443
 
 logging:
   level:
     root: INFO
 
-
 SANITY_DATASET: "ba-brev"
 
-PDL_URL: http://pdl-api-q1.pdl
+PDL_URL: https://pdl-api-q1.dev-fss-pub.nais.io
 PDL_SCOPE: api://dev-fss.pdl.pdl-api-q1/.default
 
 NORG2_BASE_URL: https://app-q1.adeo.no/norg2
 FAMILIE_BREV_API_URL: https://familie-brev.dev.intern.nav.no
```

Url til appene som kjører på GCP kan endres til http://<app>
```suggestion
FAMILIE_BREV_API_URL: http://familie-brev
```